app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ReservationsController < ApplicationController
+  def create
+    myhash = params.require(:reservation).permit(:city, :date, :item)
+    myitem = Item.where(name: myhash['item']).first
+    new_reserv = Reservation.new(city: myhash['city'], date: myhash['date'], item: myitem)
+    new_reserv.user = current_user
```

Comment:
Great approach with the hash to hold the params!

Reply:
The way I found to split it